### ISSUE TITLE
Misspelled "declare" in instructions

### ISSUE THREAD

**Issue by orangemooncreative** [freeCodeCamp/freeCodeCamp#14527]
Challenge [mutate-an-array-declared-with-const](http://beta.freecodecamp.com/en/challenges/es6/mutate-an-array-declared-with-const) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/57.0.2987.133 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

```js


const s = [ 5, 7, 2 ];
// change code below this line
s = [2, 5, 7];
// change code above this line
// Test your code
console.log(s);

```

There is a spelling error on this challenge. 

In the instructions, declared is spelled incorrectly(see below):

An array is delcared as const s = [5, 7, 2]. Change the array to [2, 5, 7].

**Comment by Manish-Giri** [comment]
@orangemooncreative Thanks for letting us know.

@erictleung good for `first-timers-only`?

**Comment by erictleung** [comment]
@orangemooncreative thanks for the issue! Yep, that is a typo.

@Manish-Giri looks like a good one for it! :+1:
 
The line that needs to be changed is [here](https://github.com/freeCodeCamp/freeCodeCamp/blob/1be73ac112ce495edb016fcc5036d8f70a9dda0d/seed/challenges/02-javascript-algorithms-and-data-structures/es6.json#L148). Please first read the [contributing guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before taking care of this issue. And feel free to visit the [Contributors Help](https://gitter.im/FreeCodeCamp/HelpContributors) chat room if you have any questions about helping. We're there to help.

**Comment by GunvantSr** [comment]
Hey Folks, i'm new here Please help me. i just corrected the typo. but i've no idea what to do next :(

**Comment by raisedadead** [comment]
Fixed via #14531
